web/src/components/Layout/Navigation/Navigation.tsx

```diff
@@ -16,12 +18,19 @@ export type Page = {
 
 export default function Navigation({ menu }: NavigationProps) {
 	return (
-		<header className="sidebar">
-			<nav className="flex flex-col items-center justify-between h-full">
-				<a className="w-3/4" href="/">
-					<img src="/vim.svg" alt="Vim Logo" />
-				</a>
-				<ul className="w-full flex flex-col gap-2">
+		<header className="h-full rounded-[35px] pt-8 bg-gray-5">
```

Comment:
`rounded-[35px]` => do we have standards for the border radius as well? If so use it. Otherwise we should add some.